.kitchen.yml

```diff
@@ -38,6 +40,24 @@ suites:
           elasticsearch:
             config:
               network.host: 0.0.0.0
+            plugins:
+              lang-python: lang-python
+  - name: version-5.0-with-pillar-set
```

Comment:
You should merge this with `version-5.0` since the pillar is already used when using version 5.